src/entities/activity/ui/items/CheckboxItem/PortraitGrid.tsx

```diff
@@ -0,0 +1,59 @@
+import RadioGroup from '@mui/material/RadioGroup';
+
+import { PortraitCheckboxOption } from './PortraitCheckboxOption';
+import { CheckboxItem } from '../../../lib/types';
+
+import Box from '~/shared/ui/Box';
+import { useCustomMediaQuery } from '~/shared/utils';
+
+type Props = {
+  options: CheckboxItem['responseValues']['options'];
+  itemId: string;
+  values: string[];
+
+  onValueChange: (value: string, isNoneAbove: boolean) => void;
+  replaceText: (value: string) => string;
+  isDisabled: boolean;
+};
+
+export const PortraitGrid = (props: Props) => {
+  const { itemId, options, onValueChange, isDisabled, replaceText, values } = props;
+
+  const { lessThanSM } = useCustomMediaQuery();
+
+  return (
+    <RadioGroup name={`${itemId}-radio portait-mode`}>
+      <Box
+        display="flex"
+        flex="1"
+        gap="16px"
+        justifyContent="flex-start"
+        flexWrap="wrap"
+        width={lessThanSM ? '320px' : undefined}
+        alignSelf={lessThanSM ? 'center' : undefined}
+      >
```

Comment:
But can you use the grid layout though? https://css-tricks.com/snippets/css/complete-guide-grid/

Reply:
Done